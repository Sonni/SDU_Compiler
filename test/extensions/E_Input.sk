var a: int;
var b: bool;

a << in;
b << in;

write a;    # Should output input to a
write b;    # On input != 0, ouput is False, otherwise True