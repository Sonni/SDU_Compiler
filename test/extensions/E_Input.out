Output first input
Output second input
